File: 02.JS-Front-End-Arrays-And-Strings-Lab/count_string_occurences.py
def count_strings(some_text, some_word):
    words_list = some_text.split(" ")
    counter = 0

    for word in words_list: 
        if word == some_word:
            counter += 1
        
    print(counter)

count_strings('This is a word and it also is a sentence', 'is')

File: 02.JS-Front-End-Arrays-And-Strings-Lab/countStringOccurences.js
function countStrings(someText, someWord) {
    const splitArray = someText.split(" ")
    let counter = 0

    for (const element of splitArray) {
        if (element === someWord) {
            counter += 1
        }
    }
    console.log(counter)
}

countStrings('This is a word and it also is a sentence', 'is')